### PR TITLE
CORE-17843 Flaky test FlowMapperServiceIntegrationTest.testStartRPCDuplicatesAndCleanup()

### PR DESCRIPTION
**Pull request by mbrkic-r3**
- Fixed stopping event mediator.
- FlowMapperServiceIntegrationTest  modified  to work with event mediator for flow event processing.

### CODE REVIEW

**Review by JamesHR3 — DISMISSED** [review]

**Review by thiagoviana — DISMISSED** [review]

**Review by vkolomeyko — DISMISSED** [review]
LGTM

**Comment by driessamyn** [comment]
Should this go straight to `release` ?

File: libs/messaging/messaging-impl/src/main/kotlin/net/corda/messaging/mediator/MultiSourceEventMediatorImpl.kt [hunk 1]
```diff
@@ -113,11 +123,13 @@ class MultiSourceEventMediatorImpl<K : Any, S : Any, E : Any>(
...
             }
         }
+        running.set(false)
     }
 
     private fun onSerializationError(event: ByteArray) {
-        log.debug { "Error serializing [$event] "}
-        TODO("Not yet implemented")
+        // TODO CORE-17012 Subscription error handling (DLQ)
+        log.error("Failed to deserialize event")
```
Comment (LWogan) [inline libs/messaging/messaging-impl/src/main/kotlin/net/corda/messaging/mediator/MultiSourceEventMediatorImpl.kt]: error logs are for system fatal errors only. we already log the serialization error at warn level in the deserializer

**Review by LWogan — DISMISSED** [review]
Happy for this to bemerged despite my comment as the test failure is blocking prs from merging to the perf branch

File: applications/workers/workers-smoketest/src/smokeTest/kotlin/net/corda/applications/workers/smoketest/flow/ConfigurationChangeTest.kt [hunk 2]
```diff
@@ -32,6 +33,7 @@ import java.util.UUID
 // their patterns are DOWN - CORE-8015
 @Order(Int.MAX_VALUE)
 @TestInstance(Lifecycle.PER_CLASS)
+@Disabled("Temporarily disabled, will be re-enabled as part of PR #4899")
```
Comment (vkolomeyko) [inline applications/workers/workers-smoketest/src/smokeTest/kotlin/net/corda/applications/workers/smoketest/flow/ConfigurationChangeTest.kt]: This test can actually be re-enabled as proven by: #4921

Comment (mbrkic-r3) [reply applications/workers/workers-smoketest/src/smokeTest/kotlin/net/corda/applications/workers/smoketest/flow/ConfigurationChangeTest.kt]: re-enabled

File: libs/messaging/messaging-impl/src/main/kotlin/net/corda/messaging/mediator/MultiSourceEventMediatorImpl.kt (hunk 1 above)
Comment (mbrkic-r3) [reply libs/messaging/messaging-impl/src/main/kotlin/net/corda/messaging/mediator/MultiSourceEventMediatorImpl.kt]: fixed

**Review by driessamyn — DISMISSED** [review]
LGTM

**Review by driessamyn — APPROVED** [review]
LGTM

**Pull request merged**